openassessment/xblock/test/test_submission.py

```diff
@@ -360,6 +360,99 @@ def test_get_student_username(self, xblock):
         resp = json.loads(resp.decode('utf-8'))
         self.assertEqual(resp['username'], 'UserName1')
 
+    @staticmethod
+    def setup_mock_team(xblock):
+        """ Enable teams and configure a mock team to be returned from the teams service
+
+            Returns:
+                the mock team for use in test validation
+        """
+
+        xblock.xmodule_runtime = Mock(
+            course_id='test_course',
+            anonymous_student_id='r5'
+        )
+
+        mock_team = {'team_id': 'rs-04',
+                     'team_name': 'Red Squadron',
+                     'team_usernames': ['Red Leader', 'Red Two', 'Red Five'],
+                     'team_url': 'rebel_alliance.org'}
+
+        xblock.teams_enabled = True
+        xblock.team_submissions_enabled = True
+        xblock.has_team = Mock(return_value=True)
+        xblock.get_team_info = Mock(return_value=mock_team)
+        xblock.get_anonymous_user_ids_for_team = Mock(return_value=['rl', 'r5', 'r2'])
+
+        return mock_team
+
+    @scenario('data/basic_scenario.xml', user_id='Red Five')
+    def test_team_submission(self, xblock):
+        """ If teams are enabled, a submission by any member should submit for each member of the team """
+
+        # given a learner is on a team
+        mock_team = self.setup_mock_team(xblock)
+
+        # when the learner submits an open assessment response
+        response = self.request(
+            xblock, 'submit', self.SUBMISSION, response_format='json')
```

Comment:
Also tiny nit (which I wont' repeat below): put the closing paren on its own line.